CHANGELOG.md

```diff
@@ -18,12 +18,19 @@ All notable changes to this project will be documented in this file.
 - CKKS: uniformized the arguments' position for all methods of the `Encoder` interface.
 - CKKS: renamed `Encoder.EncodeNTT/New` to `Encoder.Encode/New`.
 - CKKS: renamed `Scale` to `DefaultScale` in `Parameters` and `ParametersLiteral`.
-- CKKS: renamed `Encoder.EncodeDiagMatrixBSGSAtLvl` to `Encoder.EncodeDiagMatrixBSGS`.
+- CKKS: renamed `PtDiagMatrix` to `LinearTransform`
+- CKKS: changed constructors of `LinearTransform` structs from `Encoder.EncodeDiagMatrix*` to `NewLinearTransform*`.
 - CKKS: added `EncodeSlotsQP` to encode on `rlwe.PolyQP` and create custom `PtDiagMatrix`.
 - CKKS: `GetPrecisionStats` now also accepts `[]float64`.
 - CKKS: `Approximate` now accepts `func(float64)float64` and `func(complex128)complex128`
 - CKKS: approximation intervals are now `float64` instead of `complex128`.
+- CKKS: added `PolynomialVector` struct, which allows to evaluate multiple different polynomials on the same ciphertext with `Evaluator.EvaluatePoly`.
```

Comment:
There is something I don't get about why we need a new type for that and not simply a new Evaluator method, which we should probably discuss live.

Reply:
Have `Evaluator.EvaluatePoly` and `Evaluator.EvaluatePolyVector` as two API methods.

Reply:
Resolved